packages/rum-core/src/error-logging/stack-trace.js

```diff
@@ -91,7 +89,21 @@ function normalizeFunctionName(fnName) {
   return fnName
 }
 
-export function createStackTraces(errorEvent) {
+function isValidStackTrace(stackTraces) {
+  if (stackTraces.length === 0) {
+    return false
+  }
+
+  if (stackTraces.length === 1) {
+    const stackTrace = stackTraces[0]
+
+    return 'lineNumber' in stackTrace
+  }
+
+  return true
+}
+
+export function createStackTraces(stackParser, errorEvent) {
```

Comment:
Is it correct that we are now passing stackParser here in order to mock this library when testing? :)

Reply:
yes, that's right! 🎉 . In this particular scenario the option of injecting the dependency was the simplest strategy for making the testing experience smooth.

Reply:
Any reason why you didnt go with mocking `error-stack-parser` using spy instead?

Reply:
Unfortunately, with jasmine we cannot spy properly a third-party (node_modules one) dependency 😢